addons/hr_expense/models/hr_expense.py

```diff
@@ -355,7 +355,7 @@ def _compute_is_editable(self):
             managers = (
                 expense.manager_id
                 | employee.expense_manager_id
-                | employee.department_id.manager_id.user_id
+                | employee.sudo().department_id.manager_id.user_id
```

Comment:
This would result in a recordset that is or isn't sudoed (I actually do not now but don't like it to be a possibility arising from future changes)
so if you can .sudo(self.env.su) after the user_id (if the user can read the users) then it'd be better